### PR TITLE
Chore/#140474061/integrate all badges and coverall to project repository

### PR DESCRIPTION
**Pull request by tech-lady-zz**
#### What does this PR do?

This PR is meant to update the badges with the project repository and attach coveralls.

#### Description of Task to be completed:
We want contributors to see updated steps for contributing to our application and the badges pointed to our repositories on CI, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
+[![bitHound Overall Score](https://www.bithound.io/github/andela/jiayou-rc/badges/score.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![bitHound Dev Dependencies](https://www.bithound.io/github/andela/jiayou-rc/badges/devDependencies.svg)](https://www.bithound.io/github/andela/jiayou-rc/develop/dependencies/npm) [![bitHound Code](https://www.bithound.io/github/andela/jiayou-rc/badges/code.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![Code Climate](https://codeclimate.com/github/andela/jiayou-rc/badges/gpa.svg)](https://codeclimate.com/github/andela/jiayou-rc) [![Circle CI](https://circleci.com/gh/andela/jiayou-rc.svg?style=svg)](https://circleci.com/gh/andela/jiayou-rc) 
+[![Coverage Status](https://coveralls.io/repos/github/andela/jiayou-rc/badge.svg?branch=develop)](https://coveralls.io/github/andela/jiayou-rc?branch=develop)
```
Comment (rowlandekemezie) [inline README.md]: We can allow coverage status on all branches. I don't think we need `branch=develop` appended to the status report

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "faker": "^3.1.0",
     "fibers": "^1.0.14",
     "font-awesome": "^4.6.3",
+    "google-libphonenumber": "^2.0.11",
```
Comment (rowlandekemezie) [inline package.json]: What's this dependency for?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
+[![bitHound Overall Score](https://www.bithound.io/github/andela/jiayou-rc/badges/score.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![bitHound Dev Dependencies](https://www.bithound.io/github/andela/jiayou-rc/badges/devDependencies.svg)](https://www.bithound.io/github/andela/jiayou-rc/develop/dependencies/npm) [![bitHound Code](https://www.bithound.io/github/andela/jiayou-rc/badges/code.svg)](https://www.bithound.io/github/andela/jiayou-rc) [![Code Climate](https://codeclimate.com/github/andela/jiayou-rc/badges/gpa.svg)](https://codeclimate.com/github/andela/jiayou-rc) [![Circle CI](https://circleci.com/gh/andela/jiayou-rc.svg?style=svg)](https://circleci.com/gh/andela/jiayou-rc)
```
Comment (rowlandekemezie) [inline README.md:3]: Circle CI report status doesn't show on the branch. please, it's important for us to have all the branches showing their status based on the last build. Why are you pointing `bithound.io` to develop branch

File: .meteor/versions [hunk 4]
```diff
@@ -131,6 +134,9 @@ ongoworks:security@2.0.1
 ordered-dict@1.0.8
 percolate:migrations@0.9.8
 practicalmeteor:chai@2.1.0_1
+practicalmeteor:loglevel@1.2.0_2
```
Comment (rowlandekemezie) [inline .meteor/versions:137]: Curious if we need all these dependencies or they were meant for testing purposes and you forgot to clean them up

**Review by rowlandekemezie — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (tech-lady-zz) [reply package.json]: I didn't add that intentionally. It must have been a mistake while working with the multivendor task

File: README.md (hunk 1 above)
Comment (tech-lady-zz) [reply README.md]: Oh okay

File: README.md (hunk 3 above)
Comment (tech-lady-zz) [reply README.md:3]: Okay, I will fix them now

**Review by rowlandekemezie — APPROVED** [review]
LGTM

**Pull request merged**